build.gradle

```diff
@@ -682,6 +682,7 @@ clusters.each { name ->
             if (securityEnabled) {
                 plugin(provider(securityPluginOld))
                 cliSetup("opensearch-security/install_demo_configuration.sh", "-y")
```

Comment:
@DarshitChanpura how is this being used? How do we pass in "admin" as the password here? `-t`?

Reply:
we don't pass it here. 

We have to export it as a separate command.

And if you want to use admin as password you can pass in option `-t` in addition to `-y`